nmostesting/suites/IS0702Test.py

```diff
@@ -481,6 +485,222 @@ def test_05(self, test):
+                                                     "parameter within the same device"
+                                                     .format(found_sender["id"]))
+                            if device_broker_protocol is None:
+                                device_broker_protocol = sender_broker_protocol
+                            else:
+                                if device_broker_protocol != sender_broker_protocol:
+                                    return test.FAIL("Sender {} does not have the same broker_protocol "
+                                                     "parameter within the same device"
+                                                     .format(found_sender["id"]))
+                            if device_broker_authorization is None:
+                                device_broker_authorization = sender_broker_authorization
+                            else:
+                                if device_broker_authorization != sender_broker_authorization:
+                                    return test.FAIL("Sender {} does not have the same broker_authorization "
+                                                     "parameter within the same device"
+                                                     .format(found_sender["id"]))
+                        except KeyError as e:
+                            return test.FAIL("Sender {} parameters do not contain expected key: {}"
+                                             .format(found_sender["id"], e))
+                    else:
+                        return test.FAIL("Source {} has no associated IS-05 sender".format(source_id))
+            if found_senders:
+                return test.PASS()
+            else:
+                return test.UNCLEAR("Not tested. No MQTT sender resources found.")
+        else:
+            return test.UNCLEAR("Not tested. No resources found.")
+
+    def test_07(self, test):
+        """Check MQTT appropriate messages are published on the MQTT broker for each sender"""
+
+        # Gather the possible brokers and senders which can be subscribed to
+        broker_senders = self.get_mqtt_broker_senders(test)
+        warning = None
+        if len(broker_senders) > 0:
+            target_brokers = {}
+            for broker_params in broker_senders:
+                target_brokers[broker_params] = MQTTClientWorker(
+                    broker_params.host,
+                    broker_params.port,
+                    broker_params.protocol == "secure-mqtt",
+                    CONFIG.MQTT_USERNAME,
+                    CONFIG.MQTT_PASSWORD)
+            for broker_params in broker_senders:
+                target_brokers[broker_params].start()
+
+            # Give each MQTT client a chance to start and connect to the broker
+            start_time = time.time()
+            while time.time() < start_time + CONFIG.WS_MESSAGE_TIMEOUT:
```

Comment:
Let's either rename that constant or add another one for MQTT.

Reply:
yes, meant to do that